var/spack/repos/builtin/packages/intel-oneapi-advisor/package.py

```diff
@@ -9,16 +9,24 @@
 
 
 class IntelOneapiAdvisor(IntelOneApiPackage):
```

Comment:
Then we can use it like:
```suggestion
@IntelOneApiPackage.license_footer
class IntelOneapiAdvisor(IntelOneApiPackage):
```